packages/vats/src/vat-bank.js

```diff
@@ -361,6 +377,26 @@ export function buildRootObject() {
               proposedName,
             }),
           );
+
+          if (nameAdmin) {
+            // publish settled issuer identity
+            void Promise.all([kit.issuer, E(kit.brand).getDisplayInfo()]).then(
```

Comment:
why resolve `kit.issuer` when it's coming through `getDisplayInfo()` too? 

They should be the same, right? If there's a good reason to get them both, and they are identical, please assert that as a check and as documentation.

Reply:
`getDisplayInfo()` doesn't give an issuer. Just decimals and assetKind

Reply:
derp, misread the indentation.